index.html

```diff
@@ -54,8 +54,204 @@ <h4 class="social-media-intro">LET’S CONNECT</h4>
+            <img src="cardTwoImage.svg" alt="" class="card-img">
+
+        <h2 class="project-title">Multi-Post Stories</h2>
+
+        <ul class="workInfo">
+            <li class="workInfoDetails first-work"><a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a> <img class="counter" src="Counter.svg" alt=""> </li>
+            <li class="workInfoDetails"><a class="workInfoDetailsLink" href="#">Back End Dev</a> <img class="counter" src="Counter.svg" alt=""> </li>
+            <li class="workInfoDetails"><a class="workInfoDetailsLink" href="#">2015</a></li>
+        </ul>
+
+        <p class="details-sec">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="tag-btn-container">
+            <li class="tag-btn"> <button type="button"  class="btn-small">html</button> </li>
+            <li class="tag-btn"> <button type="button"  class="btn-small">css</button> </li>
+            <li class="tag-btn"> <button type="button"  class="btn-small">javascript</button></li>
+        </ul>
+
+        <button type="button"  class="btnNormal">See Project</button>
+            
+        </div>
+
+        <!-- card ends here -->
+
+        <!-- single card start here -->
+        
+        <div class="card third-card"> 
+            <img src="cardThreeImage.svg" alt="" class="card-img">
+
+        <h2 class="project-title">Tonic</h2>
+
+        <ul class="workInfo">
+            <li class="workInfoDetails first-work"><a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a> <img class="counter" src="Counter.svg" alt=""> </li>
+            <li class="workInfoDetails"><a class="workInfoDetailsLink" href="#">Back End Dev</a> <img class="counter" src="Counter.svg" alt=""> </li>
+            <li class="workInfoDetails"><a class="workInfoDetailsLink" href="#">2015</a></li>
+        </ul>
+
+        <p class="details-sec">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="tag-btn-container">
+            <li class="tag-btn"> <button type="button"  class="btn-small">html</button> </li>
+            <li class="tag-btn"> <button type="button"  class="btn-small">css</button> </li>
+            <li class="tag-btn"> <button type="button"  class="btn-small">javascript</button></li>
+        </ul>
+
+        <button type="button" class="btnNormal">See Project</button>
+            
+        </div>
+
+        <!-- card ends here -->
+
+        <!-- single card start here -->
+        
+        <div class="card fouth-card"> 
+            <img src="cardFourImage.svg" alt="" class="card-img">
+
+        <h2 class="project-title">Multi-Post
+            Stories</h2>
+
+        <ul class="workInfo">
+            <li class="workInfoDetails first-work"><a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a> <img class="counter" src="Counter.svg" alt=""> </li>
```

Comment:
- [ ] Please fix the aligning of each project use flexbox to align each one of them 
![image](https://user-images.githubusercontent.com/74744787/164703112-8e701d8e-d2b8-4663-976a-57d2b64e8f19.png)

- Your design doesn't look like [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design)

| Your design | Figma design |
| ------------- | --------------- |
| ![image](https://user-images.githubusercontent.com/74744787/164703403-8d66592f-09d2-4aaa-9c9f-e0d8bd82f04d.png) | ![image](https://user-images.githubusercontent.com/74744787/164703481-b20054dc-fcc9-4aa2-85c7-0026cb410f63.png) |
| look at this weird doubling in the card please fix it | |